openfisca_core/tests/test_yaml.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+import pkg_resources
+import os
+from nose.tools import nottest, raises
+
+from openfisca_core.tools.test_runner import run_tests, generate_tests
+
+from openfisca_core.tests.dummy_country import DummyTaxBenefitSystem
+
+tax_benefit_system = DummyTaxBenefitSystem()
+
+openfisca_core_dir = pkg_resources.get_distribution('OpenFisca-Core').location
+yamls_tests_dir = os.path.join(openfisca_core_dir, 'openfisca_core', 'tests', 'tests_yaml')
+
+nottest(run_tests)
```

Comment:
`nosetest` browses the test files, such as this one, looking for tests to run. Without this line, it thinks `run_test` is a test because it contains the word `test`, even though it is an imported function.